lib/generators/scenic/view/templates/db/migrate/update_view.erb

```diff
@@ -4,7 +4,7 @@ class <%= migration_class_name %> < <%= activerecord_migration_class %>
     update_view <%= formatted_plural_name %>,
       version: <%= version %>,
       revert_to_version: <%= previous_version %>,
-      materialized: true
+      materialized: true<%= ", materialized_no_data: true" if materialized_no_data? %>
```

Comment:
To maintain the current style, this option should be rendered on its own line. Inserting the comma will be difficult there if we don't *always* render the `no_data` option, so I'd vote for just always rendering the option (true or false). This has the added advantage of making it more obvious the option exists.